pylibczi/tests/test_czi_file.py

```diff
@@ -43,6 +43,16 @@ def test_is_mosaic(data_dir, fname, expected):
     assert czi.is_mosaic() == expected
 
 
+@pytest.mark.parametrize("fname, expected", [
+    ('s_1_t_1_c_1_z_1.czi', (False)),
+    ('s_3_t_1_c_3_z_5.czi', (False)),
+])
+def test_destructor(data_dir, fname, expected):
+    czi = CziFile(str(data_dir / fname))
+    czi = CziFile(str(data_dir / fname))
+    #this should segfault on windows
```

Comment:
should segfault?

Reply:
I confirmed this does segfault (without the fix) added pytest-xdist and changed workflow to run pytest -n1 ...  This causes the segfault to be captured and the remaining tests run but now the github actions should work.